islandora_solr_config/islandora_solr_config.module

```diff
@@ -53,7 +53,7 @@ function islandora_solr_config_islandora_solr_secondary_display() {
       'class' => 'IslandoraSolrResultsRSS',
       'function' => 'printRSS',
       'description' => 'Show search results as RSS feed',
-      'logo' => '<img src="' . $path . '/images/rss.png" class="secondary-display-rss">',
+      'logo' => '<img src="' . $path . '/images/rss.png" class="secondary-display-rss" alt="RSS Feed">',
```

Comment:
Similar [to above](https://github.com/Islandora/islandora_solr_search/pull/283/files#r66519282), "RSS Feed" should be `t()`-wrapped.